integration-tests/kafka/pom.xml

```diff
@@ -96,6 +96,27 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>with-sasl-plaintext</id>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <systemProperties>
+                                <kafka.security.protocol>SASL_PLAINTEXT
```

Comment:
Why are these properties configured here and not in the Kafka Test Resource?

Reply:
This instructs the test resource to set up the broker to request authentication and to pass the required kafka-client properties on to quarkus.

Reply:
Implemented dedicated integration test, leaving the existing ones untouched.